src/ex1/script.js

```diff
@@ -106,21 +108,25 @@ function updatePending() {
     } else {
         pendingCounter.innerHTML = `You have ${pendingTasksAmount} pending tasks`;
     }
-    showHoseClearAllButton();
+    showHideFeaturesButtons();
 }
 
-function showHoseClearAllButton() {
+function showHideFeaturesButtons() {
     if (todos.length !== 0) {
         clearAllButton.style.display = "flex";
+        filterOption.style.display = "flex";
     } else {
         clearAllButton.style.display = "none";
+        filterOption.style.display = "none";
     }
 }
 
+// Updating pending message on each task change:
+
 // https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver:
 const config = { attributes: true, childList: true, subtree: true };
 
-const callback = function (mutationsList, observer) {
+const callback = function (mutationsList) {
```

Comment:
why do you need the observer in line 139?

Reply:
if you meant for the observer instance this is for the pending tasks counter, every change in className call updatePending() function.

if you meant for the parameter 'observer' I don't need it, it got there when I copeid it from MDN (i.e code monky)